api/v1/models/index.js

```diff
@@ -8,4 +9,10 @@ module.exports = {
 	Volunteer: require('./Volunteer'),
 	Team: require('./Team'),
 	TeamsUser: require('./TeamsUser')
+=======
```

Comment:
I'm actually surprised node hasn't caught this yet haha (it shouldn't run right?)